.github/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: ci
```

Comment:
I think for this to work it has to be in the `.github/workflows` folder.

Reply:
Whoops! It was late 😅

Reply:
I guess this will only run for the next PR, after this is merged, right?